src/version.h

```diff
@@ -9,7 +9,7 @@
  * network protocol versioning
  */
 
-static const int PROTOCOL_VERSION = 170015;
+static const int PROTOCOL_VERSION = 170040;
```

Comment:
When will this get updated to 170050?

Reply:
This will occur at the same time as the testnet activation height is set, which will be in our 4.7.0 release PR.